milner_desvars/ros-notes.md

```diff
@@ -0,0 +1,490 @@
+# ROS
```

Comment:
This file is for *this* week's assignment. Let's add the new assignment to a different pull-request. Also the rqt image shouldn't go in here.